src/main/kotlin/ink/abb/pogo/scraper/Main.kt

```diff
@@ -118,6 +118,23 @@ fun main(args: Array<String>) {
         println("No item drop policy specified, defaulting to $shouldDropItems")
     }
 
+    if(shouldDropItems){
+        HashMap(uselessItems).forEach {
+            val itemName = it.key.name.toLowerCase()
+
+            try {
+                val newValue = properties.getProperty(itemName).toInt()
```

Comment:
What's the behavior of this if the property is not specified in the config file?

Will it give an NPE, or default to 0 (which would fail the `newValue < -1` check below ... and drop all your items of the omitted type).

I think we ought to try to make sure an unchanged config file (ie, omitting all these values) has the behavior of using the default values in the code.